internal/cephfs/core/volume.go

```diff
@@ -190,33 +197,32 @@ func (s *subVolumeClient) GetSubVolumeInfo(ctx context.Context) (*Subvolume, err
 	return &subvol, nil
 }
 
-type operationState int64
-
 const (
-	unknown operationState = iota
+	unknown int64 = iota
```

Comment:
not sure why this was int64 before, as we have minimal values we should have used uint32.